docs/DafnyRef/UserGuide.md

```diff
@@ -124,12 +124,25 @@ The following commands are recognized. In each case all files listed are parsed
 - `dafny verify` -- verifies the listed files. Although the Dafny program being considered
 consists of the listed files and any included files (recursively), by default only listed files are verified.
 This is similar to using the option `/compile:0` in the old-style command-line. 
-- `dafny run` -- verifies, compiles and runs the Dafny program. The option `--no-verify` suppresses verification checks. The `-t` or `--target` option states the target platform to compile to (default is C#). 
-This is similar to the old-style options `/compile:3` or `/compile:4`.
 - `dafny translate` -- verifies the program and translates it to a source artifact, perhaps with information for
 a target language build tool, for the target platform, similar to the previous `/compile:0 '/spillTargetCode:2`.
 `--no-verify` suppresses verification checks.
 - `dafny build` -- verifies (except if `--no-verify` is used) and compiles the Dafny program, producing an executable artifact for the target system. The command has options that enable being specific about the build platform and architecture.
+- `dafny run` -- verifies, compiles and runs the Dafny program. The option `--no-verify` suppresses verification checks. The `-t` or `--target` option states the target platform to compile to (default is C#). 
+This is similar to the old-style options `/compile:3` or `/compile:4`.
```

Comment:
Can you put this at the end of the documentation? We want the new syntax to be the new standard, and mentioning an old standard inside the description can be confusing. I think it'd be better if you put this line at the end of the section `dafny run`

Reply:
I just removed these comparisons entirely.